Azure/cc-setup-azure.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description: This script will make the next flow
```

Comment:
well I am not sure about this comment, I think `usage` function just should include a short description about what the script does, and how you can use use the script, maybe it could be included in the README, what do you think?

Reply:
Well, this list does show what the script does. It doesn't have to move from here, but by putting it in the usage, you make it available to the user without requiring them to fire up an editor to see what's inside :)